__mocks__/commentCount.js

```diff
@@ -0,0 +1,13 @@
+const getCommentMock = (document, data) => {
```

Comment:
[OPTIONAL] Since the previous reviewer didn't notice, I will keep it optional. Unfortunately, your both tests are incorrect because you are not testing any functions. You should have functions for counting the number of the elements on the DOM, and when you test it, you should have call that function here and make sure your counters functions would work well. But as I said, first you need functions can be called from here.
<img width="1119" alt="Screenshot 2023-03-24 at 9 01 57 PM" src="https://user-images.githubusercontent.com/25887253/227608914-e5d40df5-9281-4de0-8e51-b70f854e3383.png">